cmd/bots/internal/discord/handlers/handlers.go

```diff
@@ -0,0 +1,129 @@
+package handlers
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/pkg/errors"
+	"nodemon/cmd/bots/internal/common"
+	"nodemon/cmd/bots/internal/common/messaging"
+	"nodemon/pkg/messaging/pair"
+)
+
+func InitDscHandlers(environment *common.DiscordBotEnvironment, requestType chan<- pair.RequestPair, responsePairType <-chan pair.ResponsePair) {
+	environment.Bot.AddHandler(func(s *discordgo.Session, m *discordgo.MessageCreate) {
+		if m.Author.ID == s.State.User.ID {
+			return
+		}
+		if m.Content == "/ping" {
```

Comment:
I can't find `/help` command here. I think it's necessary.

Reply:
https://github.com/wavesplatform/nodemon/issues/83